components/relevancy/src/relevancy.udl

```diff
@@ -8,9 +8,14 @@ interface RelevancyApiError {
 // Top-level class for the Relevancy component
 interface RelevancyStore {
     // Construct a new RelevancyStore
+    //
+    // This is non-blocking since databases are lazily opened.
     [Throws=RelevancyApiError]
     constructor(string dbpath);
 
+    // Close any open databases
+    void close_databases();
```

Comment:
Suggestion: how about we call it `close()` instead? For two reasons: don't reveal the details that the store is backed by db(s); we could add other resource cleanup to this in the future.